packages/remix-architect/binaryTypes.ts

```diff
@@ -34,7 +34,8 @@ const binaryTypes = [
   "audio/basic",
   "audio/mpeg",
   "audio/ogg",
-  "audio/wavaudio/webm",
+  "audio/wav",
+  "audio/webm",
```

Comment:
there is a binaryTypes.ts in the netlify adapter as well, might need to change it there too 👉 https://github.com/remix-run/remix/blob/main/packages/remix-netlify/binaryTypes.ts#L36